national-resources/estonia.md

```diff
@@ -0,0 +1,121 @@
+* [__Prevalence of SARS-CoV-2 IgG antibodies.__](https://covid19dataportal.ee/en/data_types/serology/data/) The determination of antibodies allowed for a retrospective assessment of the outbreak's extent and is an important tool for assessing the extent of population immunity. In Estonia, two seroepidemiological studies were conducted in the spring and late summer of 2020 (Koro-Sero-EST 1 end 2). Moving forward, the determination of antibodies was incorporated into the prevalence study.
+* [__Prevalence study.__](https://ut.ee/en/content/study-prevalence-coronavirus-estonia) The primary objectives of active surveillance were to evaluate the widespread presence of SARS-CoV-2 and the prevalence of antibodies. The first stage of the surveillance study took place from April 22nd to 29th, 2020. Adults from all over Estonia were invited to participate in the study based on a random sampling. In total, 33 stages were carried out, each involving 2000 - 2500 participants. The percentage of infected individuals fluctuated from less than 1% during the first wave to over 10% during the fourth wave. Active surveillance provided the most objective overview of the extent of virus presence in Estonia. The first stages of the prevalence study have been published by [Uusküla et al 2022](https://doi.org/10.1016%2Fj.puhe.2022.02.004).
+* [__SARS-CoV-2 sequencing__](https://kliinilinemeditsiin.ut.ee/et/sisu/eesti-sars-cov-2-taisgenoomide-jarjestamine-ja-analuus-korogeno-est-1est-2est-3) was initiated at the University of Tartu in 2020. The KoroGenoEst projects 1-3, also described as a [Showcase](https://www.infectious-diseases-toolkit.org/showcase/korogenoest), led by the University of Tartu, continued their operations up until January 2023. Over three years, a total of 8,990 samples were analyzed. From 2023 onwards, the Health Board assumed responsibility for the sequencing analysis of SARS-CoV-2.
+* [__HOIA__](https://e-estonia.com/estonias-coronavirus-app-hoia-the-product-of-a-unique-private-public-partnership/) was a COVID-tracking app launched in August 2020. The app tracked contacts by capturing Bluetooth signals from nearby phones. If the signal was close and frequent enough, anonymous codes were collected and stored on the phone. If a person with the HOIA app became infected, they could alert the app, and those who were considered to have been in close contact with that person were immediately notified. The app was used by 300 000 people but was decommissioned in May 2022.
+
+## Dashboards and visualization platforms
+
+* [koroona.ut.ee](https://koroona.ut.ee/) Multiple dashboards using open data provided by the Health Board. 
```

Comment:
Maybe make this format:
[Platform name]: blurb
So the name of the platform, then a colon, then the blurb starting with a lowercase letter.